docs/tutorials/quick-start.mdx

```diff
@@ -12,28 +12,79 @@ import ImgKabob from '@site/static/img/data-explorer-kabob.png'
+
+   <img style={{marginLeft:'15px', marginBottom: '5px', height: '40px'}} src={ImgConnectionNav} />
+
+4. **Explore your database:**  
+   Within 30 seconds of adding your connection, your database will show up. Click the `Explore` button to dive in!  
 
-Next, [add your database connection](/docs/database-connections/add-new-database-connection). _For Neon Serverless Postgres, [follow this guide](/docs/tutorials/neon-add-connection)._
+   <img style={{marginLeft:'15px', marginBottom: '5px', height: '140px'}} src={ImgDatabaseCard} />
 
-After adding your connection, visit your [Organization](/docs/organizations/organizations) page by clicking your Org name in the top nav.
+5. **Create an Issue:**  
+   - In the [Data Explorer](/docs/data-explorer/data-explorer), click the Kabob button (the three dots).  
+   - Select `Create Issue`, fill in the necessary details, and submit your first [Issue](/docs/category/issues).  
 
-<img style={{marginLeft:'15px', marginBottom: '5px', height: '40px'}} src={ImgConnectionNav} />
+   <img style={{marginLeft:'15px', marginBottom: '5px', height: '140px'}} src={ImgKabob} />
 
-Once your database shows up (usually within 30 seconds of adding your connection), click the `Explore` button.
+---
+
+## Managing Data
+
+### Proposing Changes
 
-<img style={{marginLeft:'15px', marginBottom: '5px', height: '140px'}} src={ImgDatabaseCard} />
+1. **Visit the Data Explorer:**  
+   Use the `Explore` button or navigate to 'Data Explorer' via the left-hand menu.  
 
-In the [Data Explorer](/docs/data-explorer/data-explorer), click the Kabob button (the three dots) and choose `Create Issue`.
+2. **Editing Existing Data:**  
+   - Click the `Edit` button next to the row you want to modify.  
+   - Suggest your changes, then create a **Change Request**.  
 
-<img style={{marginLeft:'15px', marginBottom: '5px', height: '140px'}} src={ImgKabob} />
+3. **Adding New Rows:**  
+   - Scroll to the bottom of the table and click `Add Row`.  
+   - Fill in the new data and submit a **Change Request**.
+
+4. **Removing Data:**  
+   - Use the `Edit` button on the relevant row, propose a removal, and submit a **Change Request**.
+
+---
 
-Fill in any additional details you like and then submit your first [Issue](/docs/category/issues).
+### What Are Change Requests? 🎉
+
+If you're familiar with GitHub Pull Requests, think of **Change Requests** as a similar workflow for making database changes.  
+- **Approval Required:** Changes are not applied to the database until a team member approves them.  
+- **Data Quality Matters:** By using **Change Requests**, Sort adopts engineering best practices to ensure your data remains accurate and reliable.
+
+---
+
+## Fun Stuff! 🎉
+
+- **Manage Workflows:** Relate Issues to Change Requests for streamlined operations.  
+- **Collaborate with Teams:** Use Sort's tools for Customer Support, Data Teams, Internal Operations, or Community Managers.
```

Comment:
The Fun Stuff section should be deleted b/c it is not actionable.

Reply:
+1 Removed